test/hotspot/jtreg/runtime/cds/appcds/jvmti/ClassFileLoadHookTest.java

```diff
@@ -96,5 +96,22 @@ public static void main(String[] args) throws Exception {
                 "ClassFileLoadHook",
                 "" + ClassFileLoadHook.TestCaseId.SHARING_ON_CFLH_ON);
         TestCommon.checkExec(out);
+
+        // JEP 483: if dumped with -XX:+AOTClassLinking, cannot use archive when CFLH
+        TestCommon.testDump(appJar, sharedClasses, useWb, "-XX:+AOTClassLinking");
+        out = TestCommon.exec(appJar,
+                "-XX:+UnlockDiagnosticVMOptions",
+                "-XX:+WhiteBoxAPI", useWb,
+                "-agentlib:SimpleClassFileLoadHook=LoadMe,beforeHook,after_Hook",
+                "-Xlog:cds",
+                "ClassFileLoadHook",
+                "" + ClassFileLoadHook.TestCaseId.SHARING_ON_CFLH_ON);
+        if (out.contains("Using AOT-linked classes: false")) {
+            // We are running with VM options that do not support -XX:+AOTClassLinking
```

Comment:
AOTClassLinking requires the ability to write the full archived module graph (FMG), so it will be disabled if jtreg is executed with
- a GC that doesn't support object heap writing, such as ZGC
- a module related option that's not compatible with FMG, such as --add-opens

I edited the comment in the test to make it clear that the JVM options affects the writing of the archive. I also modified the log to something like this:

```
Using AOT-linked classes: false (static archive: no aot-linked classes)
```